### PR TITLE
Improvements on flash message doc

### PR DESCRIPTION
**Pull request by lerox**

### CODE REVIEW

**Comment by lerox** [comment]
@javiereguiluz Hi, btw I had, but i saw that we already have an existing PR to solve the problem :)

#7554 

Thanks

**Comment by lerox** [comment]
@javiereguiluz With this code I'm getting:

```
Neither the property "flashes" nor one of the methods "flashes()", "getflashes()"/"isflashes()" or "__call()" exist and have public access in class "Symfony\Component\HttpFoundation\Session\Session"
```

Symfony 3.2.2

**Comment by javiereguiluz** [comment]
@lerox sorry, you are absolutely right!!  Luckily this will be soon merged. Thanks!

File: controller.rst [hunk 1]
```diff
@@ -427,13 +427,15 @@ read any flash messages from the session:
...
 
         {# ...or you can read and display every flash message available #}
-        {% for type, flash_messages in app.session.flashes %}
-            {% for flash_message in flash_messages %}
-                <div class="flash-{{ type }}">
-                    {{ flash_message }}
-                </div>
-            {% endif %}
-        {% endfor %}
+        {% if app.session.flashBag.keys %}
```
Comment (lerox) [inline controller.rst]: yep..

**Pull request closed without merging**